uikit-dz/uikit-dz/ViewController/Models/Service.swift

```diff
@@ -0,0 +1,60 @@
+// Service.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+final class Service {
```

Comment:
```suggestion
final class Service {
```
Service чего именно?